scripts/breaking_changes_checker/breaking_changes_tracker.py

```diff
@@ -29,76 +29,95 @@ class BreakingChangeType(str, Enum):
     REMOVED_PARAMETER_DEFAULT_VALUE_MSG = \
-        "({}): The class '{}.{}' method '{}' had default value '{}' removed from its parameter '{}' in " \
+        "The class '{}.{}' method '{}' had default value '{}' removed from its parameter '{}' in " \
         "the current version"
     REMOVED_PARAMETER_DEFAULT_VALUE_OF_FUNCTION_MSG = \
-        "({}): The publicly exposed function '{}.{}' had default value '{}' removed from its parameter '{}' in " \
+        "The publicly exposed function '{}.{}' had default value '{}' removed from its parameter '{}' in " \
         "the current version"
     CHANGED_PARAMETER_ORDERING_MSG = \
-        "({}): The class '{}.{}' method '{}' had its parameters re-ordered from '{}' to '{}' in the current version"
+        "The class '{}.{}' method '{}' had its parameters re-ordered from '{}' to '{}' in the current version"
     CHANGED_PARAMETER_ORDERING_OF_FUNCTION_MSG = \
-        "({}): The publicly exposed function '{}.{}' had its parameters re-ordered from '{}' to '{}' in " \
+        "The publicly exposed function '{}.{}' had its parameters re-ordered from '{}' to '{}' in " \
         "the current version"
     CHANGED_PARAMETER_KIND_MSG = \
-        "({}): The class '{}.{}' method '{}' had its parameter '{}' changed from '{}' to '{}' in the current version"
+        "The class '{}.{}' method '{}' had its parameter '{}' changed from '{}' to '{}' in the current version"
     CHANGED_PARAMETER_KIND_OF_FUNCTION_MSG = \
-        "({}): The function '{}.{}' had its parameter '{}' changed from '{}' to '{}' in the current version"
+        "The function '{}.{}' had its parameter '{}' changed from '{}' to '{}' in the current version"
     CHANGED_CLASS_FUNCTION_KIND_MSG = \
-        "({}): The class '{}.{}' method '{}' changed from '{}' to '{}' in the current version."
+        "The class '{}.{}' method '{}' changed from '{}' to '{}' in the current version."
     CHANGED_FUNCTION_KIND_MSG = \
-        "({}): The function '{}.{}' changed from '{}' to '{}' in the current version."
+        "The function '{}.{}' changed from '{}' to '{}' in the current version."
     REMOVED_OR_RENAMED_MODULE_MSG = \
-        "({}): The '{}' module was deleted or renamed in the current version"
+        "The '{}' module was deleted or renamed in the current version"
     REMOVED_CLASS_FUNCTION_KWARGS_MSG = \
-        "({}): The class '{}.{}' method '{}' changed from accepting keyword arguments to not accepting them in " \
+        "The class '{}.{}' method '{}' changed from accepting keyword arguments to not accepting them in " \
         "the current version"
     REMOVED_FUNCTION_KWARGS_MSG = \
-        "({}): The function '{}.{}' changed from accepting keyword arguments to not accepting them in " \
+        "The function '{}.{}' changed from accepting keyword arguments to not accepting them in " \
         "the current version"
 
+    # ----------------- General Changes -----------------
```

Comment:
Should the changes checker be a separate class instead of building on top of BreakingChangesTracker? My initial feeling is that it should be, but let me know what you think. I won't block on this, though

Reply:
Yes it should be, but will do so in the future, adding a tracking issue for that specifically: https://github.com/Azure/azure-sdk-for-python/issues/36066

I'm currently thinking that the changes tracker class should inherit from the breaking changes class given that breaking changes checking is the basis for both of these functionalities. Let me know if you have any thoughts about it :)